sw/host/opentitanlib/src/tpm/driver.rs

```diff
@@ -255,13 +260,43 @@ const MAX_RESPONSE_SIZE: usize = 4096;
 
 impl Driver for SpiDriver {
     fn read_register(&self, register: Register, data: &mut [u8]) -> Result<()> {
+        if !self.spi.supports_bidirectional_transfer()? {
+            /*
+             * SPI transport does not support bidirectional transfer.  Assume that the TPM will
+             * send 0x01 on the byte immediately following the forth and final request byte.
```

Comment:
nit: Use `//` comments.
nit: fourth (here and in the comment below).

Reply:
I guess I have written too much HyperDebug firmware in plain C recently.  Fixed comment syntax and spelling.